Dockerfile

```diff
@@ -5,7 +5,7 @@
 #
 #       If you do not use buildkit you are not going to have a good time
 #
-#       For reference: 
```

Comment:
Trailing whitespace is not cool kids

Reply:
Would it make sense to add a lint check for trailing whitespace? (similar to our existing check for tabs)

Reply:
Probably, I don’t think trailing whitespace is acceptable anywhere in this repository outside of places we can’t change it like `third_party`

Reply:
Cool, I'll do that soon then